Processen/ServiceDesk.md

```diff
@@ -22,6 +22,28 @@ Het kent de volgende subprocessen die in de documenten in de onderliggende links
 
 De stappen in het Service Desk proces zijn genummerd met een uniek nummer en hieronder in meer detail beschreven.
```

Comment:
```suggestion
De stappen in het Service Desk proces zijn genummerd met een uniek nummer en hieronder in meer detail beschreven.
```
![Proces Servicedesk](https://github.com/VNG-Realisatie/api-beheer/blob/master/Processen/ServiceDesk.jpg "Proces Servicedesk")

De stappen in het Service Desk proces zijn genummerd met een uniek nummer en hieronder in meer detail beschreven.

Reply:
Ik heb nu blijkbaar een suggestie voor een klikbare link naar het plaatje van het proces opgenomen.

Maar het lijkt me goed om de afbeelding op dezelfde pagina op te nemen.

Reply:
Ik begrijp niet goed wat je bedoelt, bij mij staat de afbeelding ook in dit document direct boven de beschrijving van alle stappen.